src/store/slices/slicePopularCategory.ts

```diff
@@ -0,0 +1,41 @@
+import { createAsyncThunk, createSlice } from "@reduxjs/toolkit";
+import {Category, Good, LOAD_STATUSES} from "../../types";
+import { getPopularCategories} from "../api";
+
+const SLICE_NAME = "popularCategories";
+
+export const fetchPopularCategories = createAsyncThunk(`${SLICE_NAME}/fetchPopularCategories`, getPopularCategories);
+
+export interface PopularGoodsStore {
+    loadStatus: string,
+    popular: {category: Category;
+    items: Good[]}[]
+}
+
+const initialState: PopularGoodsStore = {
+    loadStatus: LOAD_STATUSES.UNKNOWN,
+    popular: [{items: [],
```

Comment:
Для массивов начальное значение всегда пустой массив, не надо придумывать несуществующие данные
```suggestion
    popular: [],
```